apps/organisations/urlresolvers.py

```diff
@@ -7,41 +7,42 @@
 
 from apps.django_overwrites.urlresolvers import django_reverse
 
-from . import get_partner
+from . import get_organisation
 
-_partner_pattern_names = set()
+_organisation_pattern_names = set()
 
 
-def partner_patterns(*pattern_list):
-    """Mark the url patterns used with partners."""
+def organisation_patterns(*pattern_list):
+    """Mark the url patterns used with organisations."""
     for pattern in pattern_list:
         if isinstance(pattern, URLPattern):
-            _partner_pattern_names.add(pattern.name)
+            _organisation_pattern_names.add(pattern.name)
         elif isinstance(pattern, URLResolver):
             for url_pattern in pattern.url_patterns:
                 ns = ''
                 if pattern.app_name:
                     ns = ns + pattern.app_name + ':'
                 elif pattern.namespace:
                     ns = ns + pattern.namespace + ':'
-                _partner_pattern_names.add(ns + url_pattern.name)
+                _organisation_pattern_names.add(ns + url_pattern.name)
         else:
             raise Exception()
 
-    return url(r'^(?P<partner_slug>[-\w_]+)/',
+    return url(r'^(?P<organisation_slug>[-\w_]+)/',
                include(list(pattern_list)))
```

Comment:
if we add this explicitly ti our url list, then we don't need this anymore, right?

Reply:
Yep, also stuff for a follow up clean-up PR, ok?

Reply:
yes